fuseimg/data/ops/aug/geometry.py

```diff
@@ -57,7 +57,8 @@ def __call__(self, sample_dict: NDict, key: str, rotate: float = 0.0, translate:
         for channel in channels:
             aug_channel_tensor = aug_input[channel].numpy()
             aug_channel_tensor = Image.fromarray(aug_channel_tensor)
-            aug_channel_tensor = TTF.affine(aug_channel_tensor, angle=rotate, scale=scale, translate=translate, shear=shear)
+            aug_channel_tensor = TTF.affine(aug_channel_tensor, angle=rotate, scale=scale, 
+                                            resample=Image.BILINEAR, translate=translate, shear=shear)
```

Comment:
can you please add it to the as an argument? It's ok to set bilinear as default.